test/test_invalid_headers.py

```diff
@@ -113,6 +113,39 @@ def test_transfer_encoding_trailers_is_valid(self, frame_factory):
         request_event = events[0]
         assert request_event.headers == headers
 
+    def test_pseudo_headers_rejected_in_trailer(self, frame_factory):
+        """
+        Ensure we reject pseudo headers included in trailers
+        """
+        headers = (
+            [h for h in self.base_request_headers if h[0] != ':path'] +
+            [('trailers', 'extra-header')]
+        )
+        trailers = (
+            [(':path', '/'),
+             ('extra-header', 'values')]
+        )
```

Comment:
You don't need the external parentheses: Python does implicit line continuation in any kind of bracket.